site/docs/actions/public/L1/getSecondsToNextL2Output.md

```diff
@@ -2,7 +2,9 @@
 
 Returns how long until the next L2 output, for a given chain, is posted on L1. This is useful when waiting to prove a withdrawal.
 
-```ts
+::: code-group
+
+```ts [example.ts]
```

Comment:
we only really do this if we have another tab called like `config.ts`

Reply:
Seen this present in a lot of counterexamples like [getL2HashesForDepositTx](https://github.com/base-org/op-viem/blob/main/site/docs/actions/public/L1/getL2HashesForDepositTx.md) but will modify them all as needed.

Reply:
ah ok, sorry for the confusion then. That doc was written by someone else a while ago and I didn't QA closely. Not a huge deal either way. Thanks for eyeing the details